dist/site.webmanifest

```diff
@@ -0,0 +1,12 @@
+{
+  "short_name": "",
+  "name": "",
```

Comment:
This has something to do with creating shortcuts on mobile, right? (and I believe now that `browserconfig.xml` is related to Windows 10 in a similar fashion)...

Btw, isn't **`name`** required here?

Reply:
Yeah it is but I purposefully left it blank as I wanted to make sure that this gets filled in 'properly' by someone other than me. I knew if put something in there it would probably stay like that - so leaving it blank has created this conversation 👍 Should this be Codidact or Codidact Q&A Platform or something more descriptive.

Reply:
If you see this - I'm inclined to think we should put 'TODO' here. What do you think?

Reply:
sure - that makes sense